samples/crypto/persistent_key_usage/prj.conf

```diff
@@ -4,14 +4,9 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 
-# Enable loging using RTT and UART
+# Enable loging
```

Comment:
Typo: loging => logging (multiple files)

Reply:
Wonder how often that typo already got copy pasted, as it is in all samples :D, gonna fix it